automatic/displayfusion/displayfusion.nuspec

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>displayfusion</id>
+    <version>0.0</version>
+    <title>DisplayFusion</title>
+    <authors>Binary Fortress Software</authors>
+    <projectUrl>http://www.displayfusion.com</projectUrl>
```

Comment:
well, that's fine. it was mainly to have it consistent with the other urls for that website.